stl/inc/yvals_core.h

```diff
@@ -20,6 +20,7 @@
 #if _STL_COMPILER_PREPROCESSOR
 // _HAS_CXX20 directly controls:
 // P0020R6 atomic<float>, atomic<double>, atomic<long double>
+// P0122R7 span

```

Comment:
Nitpick: This should be listed as `P0122R7 <span>` (because it adds the whole header).

Reply:
Done